src/liquid/global/styles/theming/theming.css

```
@@ -0,0 +1,315 @@
+  --ld-thm-tea-error: var(--ld-col-rr);
+  --ld-thm-tea-primary-active: var(--ld-col-rg-900);
+  --ld-thm-tea-primary-alpha-low: var(--ld-col-rg-alpha-low);
+  --ld-thm-tea-primary-alpha-lowest: var(--ld-col-rg-alpha-lowest);
+  --ld-thm-tea-primary-focus: var(--ld-col-rg-100);
+  --ld-thm-tea-primary-highlight: var(--ld-col-rg-010);
+  --ld-thm-tea-primary-hover: var(--ld-col-rg-700);
+  --ld-thm-tea-primary: var(--ld-col-rg);
+  --ld-thm-tea-secondary-active: var(--ld-col-vg-800);
+  --ld-thm-tea-secondary-focus: var(--ld-col-vg-200);
+  --ld-thm-tea-secondary-highlight: var(--ld-col-vg-010);
+  --ld-thm-tea-secondary-hover: var(--ld-col-vg-400);
+  --ld-thm-tea-secondary: var(--ld-col-vg);
+  --ld-thm-tea-success-active: var(--ld-col-vg-800);
+  --ld-thm-tea-success-focus: var(--ld-col-vg-200);
+  --ld-thm-tea-success-highlight: var(--ld-col-vg-010);
+  --ld-thm-tea-success-hover: var(--ld-col-vg-400);
+  --ld-thm-tea-success: var(--ld-col-vg);
+  --ld-thm-tea-warning-active: var(--ld-col-vy-800);
+  --ld-thm-tea-warning-focus: var(--ld-col-vy-100);
+  --ld-thm-tea-warning-highlight: var(--ld-col-vy-010);
+  --ld-thm-tea-warning-hover: var(--ld-col-vy-400);
+  --ld-thm-tea-warning: var(--ld-col-vy);
+  --ld-thm-error-active: var(--ld-thm-ocean-error-active);
+  --ld-thm-error-focus: var(--ld-thm-ocean-error-focus);
+  --ld-thm-error-highlight: var(--ld-thm-ocean-error-highlight);
+  --ld-thm-error-hover: var(--ld-thm-ocean-error-hover);
+  --ld-thm-error: var(--ld-thm-ocean-error);
+  --ld-thm-primary-active: var(--ld-thm-ocean-primary-active);
+  --ld-thm-primary-alpha-low: var(--ld-thm-ocean-primary-alpha-low);
+  --ld-thm-primary-alpha-lowest: var(--ld-thm-ocean-primary-alpha-lowest);
+  --ld-thm-primary-focus: var(--ld-thm-ocean-primary-focus);
+  --ld-thm-primary-highlight: var(--ld-thm-ocean-primary-highlight);
+  --ld-thm-primary-hover: var(--ld-thm-ocean-primary-hover);
+  --ld-thm-primary: var(--ld-thm-ocean-primary);
+  --ld-thm-secondary-active: var(--ld-thm-ocean-secondary-active);
+  --ld-thm-secondary-focus: var(--ld-thm-ocean-secondary-focus);
+  --ld-thm-secondary-highlight: var(--ld-thm-ocean-secondary-highlight);
+  --ld-thm-secondary-hover: var(--ld-thm-ocean-secondary-hover);
+  --ld-thm-secondary: var(--ld-thm-ocean-secondary);
+  --ld-thm-success-active: var(--ld-thm-ocean-success-active);
+  --ld-thm-success-focus: var(--ld-thm-ocean-success-focus);
+  --ld-thm-success-highlight: var(--ld-thm-ocean-success-highlight);
+  --ld-thm-success-hover: var(--ld-thm-ocean-success-hover);
+  --ld-thm-success: var(--ld-thm-ocean-success);
+  --ld-thm-warning-active: var(--ld-thm-ocean-warning-active);
+  --ld-thm-warning-focus: var(--ld-thm-ocean-warning-focus);
+  --ld-thm-warning-highlight: var(--ld-thm-ocean-warning-highlight);
+  --ld-thm-warning-hover: var(--ld-thm-ocean-warning-hover);
+  --ld-thm-warning: var(--ld-thm-ocean-warning);
+}
+.ld-theme-ocean,
+[class*='ld-theme'] .ld-theme-ocean {
```

I believe we don't even need this anymore.

```suggestion
.ld-theme-ocean {
```